docs/rules/0192/no-html.md

```diff
@@ -0,0 +1,75 @@
+---
+rule:
+  aip: 192
+  name: [core, '0192', no-html]
+  summary: Comments must not use raw HTML
+permalink: /192/no-html
+redirect_from:
+  - /0192/no-html
+---
+
+# No HTML in comments
+
+This rule enforces that every descriptor in every proto file does not use raw
+HTML in comments, as mandated in [AIP-192][].
+
+## Details
+
+This rule looks at each descriptor in each proto file (exempting oneofs and the
+file itself) and tries to pick up "HTML smell", and complains if it thinks it
+found HTML.
+
+**Note:** This lint rule uses a regular expression to look for HTML, which is a
```

Comment:
Consider to rephrase it to "**Note:** May miss some complex HTML. ..."

Reply:
_Stet._